CafePrices/Podfile

```diff
@@ -0,0 +1,25 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'CafePrices' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for CafePrices
+  
+pod 'Firebase/Database'
+pod 'Firebase/Auth'
+pod 'Firebase/Storage'
```

Comment:
what is the difference between Firebase/Storage and FirebaseUI/Storage, are both of them necessary?

Reply:
Firebase is a platform that you use to build web and mobile applications. It consists of a suite of cloud services and a set of SDKs (and in some cases REST APIs) to access those services.

FirebaseUI is a set of libraries that build on top of the Firebase SDKs to make binding to user interface elements easier.

Specifically: FirebaseUI for Android and for iOS wrap the Database and Authentication SDKs of Firebase to make it easy to use the services in Android and iOS applications.

FirebaseUI for web is a newer entrant to the field and only wraps the Authentication SDK. To make it easy using the database in your web app, look for a dedicated library for your preferred web platform: AngularFire, AngularFire2, ReactFire, EmberFire. The names probably speak for themselves in what web library they target.

Reply:
ok lets let it there